docs/data-sources/container_registry_locations.md

```diff
@@ -0,0 +1,28 @@
+---
+subcategory: "Container Registry"
+layout: "ionoscloud"
+page_title: "IonosCloud : ionoscloud_container_registry_locations"
+sidebar_current: "docs-ionoscloud_container_registry_locations"
+description: |-
+Get list of Container Registry Locations
+---
+
+# ionoscloud_container_registry_locations
+
+The **Container Registry Token data source** can be used to get a list of Container Registry Locations
```

Comment:
@cristiGuranIonos I think that here it should be:
`The **Container Registry Locations data source** can be used to get a list of Container Registry Locations.`